src/components/TodoList.tsx

```diff
@@ -0,0 +1,23 @@
+import { TodoData } from '../types/TodoData';
+import { Todo } from './Todo';
+
+interface TodoListProps {
+  todos: TodoData[];
+  onTodoDelete: (todoId: number) => Promise<void>;
+}
+
+export const TodoList = ({ onTodoDelete, todos }: TodoListProps) => {
+  return (
+    <>
+      {todos.map((todo) => (
+        <Todo
+          key={todo.id}
+          todo={todo}
+          isTempTodo={false}
```

Comment:
I think you don't even have to pass it explicitly here if you want it.to be false (it will be undefined but it will work the same)